src/frontend/efiling-demo/nginx.conf

```diff
@@ -11,21 +11,6 @@ server {
   location ~* \.env-config\.js$ {
     expires -1;
     add_header 'Cache-Control' 'no-store, no-cache, must-revalidate, proxy-revalidate, max-age=0';
-    add_header X-Content-Type-Options "nosniff" always;
-    add_header X-XSS-Protection "1; mode=block";
-    add_header X-Content-Type-Options "nosniff" always;
-    add_header Cache-Control "no-store";
-    add_header "Pragma" "no-cache";
-    add_header Content-Security-Policy "default-src 'self';";
-    add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
-  }
-
-  location /images {
-    deny all;
-  }
-
-  location /static {
-    deny all;
```

Comment:
Lets keep the nginx changes separate from this PR so we can track it better. 
Also, I think Taylor came up with a different fix.

Reply:
Ya I just saw that, thanks:)